doc/source/whatsnew/v1.1.0.rst

```diff
@@ -140,7 +140,7 @@ Groupby/resample/rolling
 Reshaping
 ^^^^^^^^^
 
--
+- Bug in :meth:`DataFrame._reduce` not returning subclassed data type effecting all numeric and boolean reduction methods. (:issue:`25596`)
```

Comment:
this is not a public method, can you update with something a user would want to know, e.g. ``DataFrame.sum()`` etc.

Reply:
List all 12 methods then?

Reply:
I'm working on a more comprehensive review of subclassing. How about I just add that to this PR and list all the methods? It'll be ~20 methods, but it'll get it all out of the way in one go.

Reply:
u can list if u want or just sat reduction methods